pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -85,7 +81,8 @@ func (n *VMNetworkConfigurator) SetupPodNetworkPhase1(pid int) error {
 }
 
 func (n *VMNetworkConfigurator) SetupPodNetworkPhase2(domain *api.Domain) error {
-	nics, err := n.getNICs()
+	var launcherPID *int
+	nics, err := n.getNICs(newPodNICWithoutInfraConfigurator, launcherPID)
```

Comment:
why not just passing `nil` instead of `launcherPID`?

Reply:
to denote that the argument is the launcherPID, golang has no named arguments.